src/bin/coverage.rs

```diff
@@ -255,19 +258,15 @@ fn main() -> Result<(), failure::Error> {
             // Not much to do, try to parse the file and report the result.
             parse_file_with(&file, |result| {
                 let mut ambiguity_result = Ok(());
-                match result {
-                    Ok(handle) | Err(parse::ParseError::TooShort(handle)) => {
-                        ambiguity_result = ambiguity_check(handle);
-
-                        if let Some(out_path) = graphviz_forest {
-                            handle
-                                .parser
-                                .sppf
-                                .dump_graphviz(&mut fs::File::create(out_path).unwrap())
-                                .unwrap();
-                        }
+                if let Ok(Ok(handle)) = result {
```

Comment:
This doesn't handle the partial results anymore.

Reply:
As above, `ParseError<Span, Range<Span>>` doesn't give us a Handle to call `ambiguity_check` on, just `{ partial: Option<Range<Span>>, at: Span, expected }`. That's what I've got to work with currently, anyway.

Reply:
Wait, hang on, `partial` really has that type?! It should be an owned handle with the same type. Gotta fix this in `gll` I guess...

This could explain why the tests I added have the span range output and not what I wanted...

Reply:
The ambiguity is just ignored for the partial results anyway